addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
```

Comment:
To be more Emberish, plus, I think it's more visually appealing to have fewer handlebars. One set surrounding the `input` helper opposed to 4 surrounding the attrs and actions.